lib/driver.js

```diff
@@ -172,11 +171,11 @@ class XCUITestDriver extends BaseDriver {
 
     if (this.opts.browserName) {
       log.info('Safari test requested');
-      this.safari = true;      
+      this.safari = true;
       this.opts.app = undefined;
       this.opts.processArguments = this.opts.processArguments || {};
       if (this.isRealDevice()) {
-        this.opts.bundleId = SAFARI_LAUNCHER_BUNDLE;
+        this.opts.bundleId = SAFARI_BUNDLE_ID;
         this._currentUrl = this.opts.safariInitialUrl || `http://appium.io`;
         this.opts.processArguments.args = [this._currentUrl];
```

Comment:
I've just followed the guides from WebDriverAgent, The bundleId is always is bundle id of Safari.
 I don't know we have plan to support for other browsers (aka chrome) or not?

```
If application is already installed on device you can start it using bundleId parameter:

curl -X POST $JSON_HEADER \
-d "{\"desiredCapabilities\":{\"bundleId\":\"com.apple.mobilesafari\"}}" \
$DEVICE_URL/session
It is also possible to request to install application before launching it by adding app parameter:

curl -X POST $JSON_HEADER \
-d "{\"desiredCapabilities\":{\"bundleId\":\"com.apple.mobilesafari\", \"app\":\"[host_path]/magicapp.app\"}}" \
$DEVICE_URL/session
```